swiss-ng-api/berlin-group-documentation/current.yaml

```diff
@@ -10773,18 +11009,23 @@ components:
         type: string
 
 
-    bookingStatus:
+    bookingStatusGeneric:
```

Comment:
Relevant Change: bookingStatusGeneric renamed bookingStatus
Adoption approach: Follow the change